src/SignalR/clients/ts/signalr/README.md

```diff
@@ -1,4 +1,4 @@
-JavaScript and TypeScript clients for SignalR for ASP.NET Core
+JavaScript and TypeScript clients for SignalR for ASP.NET Core and Azure SignalR Service
```

Comment:
I feel like this is implied though. Probably no harm in adding it but is it necessary?

Reply:
As someone who didn't know about Azure SignalR Service, it was unclear to me 😅

Reply:
Fair enough lol

Reply:
Yeah I found through a discussion with @bnb and others that if someone landed on the NPM page while searching for how to use SignalR with Functions, they hit a dead-end because the text suggests that this package only works with ASP.NET Core SignalR.

Reply:
I'm sort of with @mikaelm12 on this in that i feel like it is implied, but @bnb makes a good point **as always** so i'm also in line with @mikaelm12's flexibility. 👍